django/utils/deprecation.py

```diff
@@ -140,3 +144,25 @@ async def __acall__(self, request):
                 thread_sensitive=True,
             )(request, response)
         return response
+
+
+def adjust_stacklevel_for_warning(skip_file_prefixes):
```

Comment:
feel like we should just call `skip_file_prefixes` here something like `file`, since we're expecting people to pass in `__file__` in general.